watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination_withRO.py

```diff
@@ -94,21 +113,26 @@ def main():
     optimize_operation(m)  # unfixes specific variables for cost optimization
 
     solve(m, checkpoint="solve flowsheet after costing")
+
     display_results(m)
     display_costing(m)
 
     return m, results
 
 
-def build(include_pretreatment=False, include_dewatering=False):
+def build(
+    include_pretreatment=False,
+    include_dewatering=False,
+    include_gac=False,
+):
```

Comment:
Is it not possible to merge `dye_desalination` and `dye_desalination_withRO` based on config arguments?

Reply:
These probably can and should be merged into one flowsheet. To provide some additional context, all of these configuration options in `dye_desalination_withRO` are fairly new, and ig the original developer decided to create two separate flowsheets instead of having a `include_RO` config option.

@adam-a-a Is there a reason I'm missing as to why these two flowsheets _shouldn't_ be combined into a single flowsheet.

Reply:
Yea this is probably still relevant for later PR improvements.

Reply:
I definitely agree that these should ultimately be merged into one flowsheet, where the inclusion of RO would become another Boolean option. As @MarcusHolly pointed out, the original developer decided to break them down initially.